pkgs/applications/misc/xpad/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, autoreconfHook, pkgconfig, glib, intltool, gtk3, gtksourceview }:
+
+stdenv.mkDerivation rec {
+  name = "xpad-${version}";
+  version = "5.0.0";
+
+  src = fetchurl {
+    url = "https://launchpad.net/xpad/trunk/${version}/+download/xpad-${version}.tar.bz2";
+    sha256 = "02yikxg6z9bwla09ka001ppjlpbv5kbza3za9asazm5aiz376mkb";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig ];
+
+  buildInputs = [ glib intltool gtk3 gtksourceview ];
+
+  autoreconfPhase = ''
+    ./autogen.sh
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Xpad is a sticky note application for jotting down things to remember.";
```

Comment:
The description should not start with the package name and should not have a period at the end. No big deal obviously but this is mentioned in CONTRIBUTING.md btw, which you checked off.

Reply:
Sorry, I needed it fast in Nixpkgs and just copied mindlessly from upstream! Will fix in a sec.